mne/io/eeglab/eeglab.py

```diff
@@ -244,7 +257,7 @@ class RawEEGLAB(_BaseRaw):
     """
     @verbose
     def __init__(self, input_fname, montage, preload=False, eog=(),
-                 verbose=None):
+                 event_id=dict(), verbose=None):
```

Comment:
No, it's terrible, I'm sorry. I'll definitely make it more straight-forward before I ask for an actual review.